apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/toggle-sidebar-button/index.tsx

```diff
@@ -17,11 +18,11 @@ import './style.scss';
 const Button = ( {
 	children,
 	...rest
-}: OriginalButton.Props & { icon?: any; iconSize?: number; showTooltip?: boolean } ) => (
+}: OriginalButton.Props & { icon?: unknown; iconSize?: number; showTooltip?: boolean } ) => (
 	<OriginalButton { ...rest }>{ children }</OriginalButton>
 );
 
-export default function ToggleSidebarButton() {
+export default function ToggleSidebarButton(): JSX.Element {
```

Comment:
Are we using `JSX.Element` for component return types, or is there something better?

Reply:
JSX.Element is perfect, the definition of a FunctionComponent is:

```
(props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
```

And in `JSX`: 

```
interface Element extends React.ReactElement<any, any> { }
```